pack/return/rttfa.json

```diff
@@ -16,6 +33,134 @@
 		"type_code": "asset",
 		"xp": 2
 	},
+	{
+		"code": "53003",
+		"cost": 1,
+		"deck_limit": 2,
+		"faction_code": "seeker",
+		"flavor": "“How do you know I’m mad?” said Alice.<br/>“You must be,” said the Cat, “or you wouldn’t have come here.”<br/>– Lewis Carroll, Alice’s Adventures in Wonderland",
+		"illustrator": "Maggie Ivy",
+		"name": "Truth from Fiction",
+		"pack_code": "rttfa",
+		"position": 3,
+		"quantity": 2,
+		"skill_intellect": 3,
+		"text": "Place 2 secrets among assets controlled by investigators at your location (3 secrets instead if there is a clue at your location).",
+		"traits": "Insight.",
+		"type_code": "event",
+		"xp": 2
+	},
+	{
+		"code": "53004",
+		"cost": 2,
+		"deck_limit": 2,
+		"faction_code": "seeker",
+		"illustrator": "Greg Bobrowski",
+		"name": "Ancient Stone",
+		"pack_code": "rttfa",
+		"position": 4,
+		"quantity": 2,
+		"skill_agility": 2,
+		"slot": "Hand",
+		"subname": "Transient Thoughts",
+		"text": "You can only include this asset in your deck by upgrading it from Ancient Stone <i>(Unidentified)</i>, and only if you have \"identified the stone\" in your Campaign Log.\nUses (X secrets). X is the number in parentheses next to \"you have identified the stone.\"\n[reaction] When you draw any number of cards, spend that many secrets: Move that many times.",
```

Comment:
\"you have identified the stone\"

looks like it's written wrong for the other ancient stones as well, i'll fix those

Reply:
Yep!